include/rocksdb/env.h

```diff
@@ -1339,253 +1345,280 @@ extern Status ReadFileToString(Env* env, const std::string& fname,
 // functionality of another Env.
 class EnvWrapper : public Env {
  public:
+  struct Target {
```

Comment:
Can you explain why we want to bring in the Target here?

Reply:
Added comments